src/main/java/com/microsoft/aad/msal4j/AadInstanceDiscoveryProvider.java

```diff
@@ -56,14 +55,35 @@ static InstanceDiscoveryMetadataEntry getMetadataEntry(URL authorityUrl,
                                                            boolean validateAuthority,
                                                            MsalRequest msalRequest,
                                                            ServiceBundle serviceBundle) {
+        String host = authorityUrl.getHost();
+
+        if (useRegionalEndpoint(msalRequest)) {
```

Comment:
nit - consider naming it to 'shouldUseRegionalEndpoint' / 'hasRegionalEndpoint' / 'isRegional' to signify that the method returns boolean.

Reply:
Changed to shouldUseRegionalEndpoint in the latest commit